<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Settings_Screen</class>
 <widget class="QWidget" name="HNS_Settings_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="GB_Radar">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>310</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:119px;
	padding-right:119px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>RADAR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="LBL_Settings_RadarMode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Normal Mode</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Settings_RadarUnits">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MPH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Settings_Radar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PB_Settings_Back">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Admin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>ADMIN</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>310</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:103px;
	padding-right:103px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>Date/Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="LBL_Settings_Date">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Monday, September 3rd, 2020 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Settings_Time">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>06:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Settings_DateTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>SETTINGS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="GB_Communications">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>310</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:59px;
	padding-right:60px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>COMMUNICATIONS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="LBL_Settings_CommMode">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Serial and Ethernet</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Settings_IPMode">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Static IP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Settings_Comm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Settings_IP_Port">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port: 5102</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PB_Diagnostics">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>DIAGNOSTIC</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>310</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:76px;
	padding-right:76px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>PAGE DEFAULTS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="LBL_Settings_LineJustification">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Center Line Justification</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Settings_PageJustification">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Center Page Justification</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Settings_PageDefaults">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Settings_DefaultFont">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Default Font: 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>310</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:90px;
	padding-right:90px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>BRIGHTNESS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="LBL_Settings_BrightMode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manual Mode</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Settings_BrightLevel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>250</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Level 4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Settings_Brightness">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GB_AuxIO">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>350</y>
     <width>310</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:114px;
	padding-right:115px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>AUX I/O</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="PB_Settings_AuxIO">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
